src/box/space.h

```diff
@@ -545,7 +545,7 @@ space_is_vinyl(const struct space *space)
  * meaning for tarantool and has predefined insert/remove triggers.
  */
 bool
-space_is_system(const struct space *space);
+space_is_system(uint32_t space_id);
```

Comment:
Judging by its name, `space_is_system()` is supposed to take a space, not id. Anyway, this function isn't very convenient to use in case we have a space. It's better to introduce a new helper function `space_id_is_system()` instead of changing the signature of `space_is_system`. This change had better be squashed into the patch that needs this function.

Reply:
> This change had better be squashed into the patch that needs this function.

Why? This an independent change.

Reply:
It's okay to leave it in a separate patch. However, come to think of it, the new function belongs in `schema_def.h`, because it only depends on `BOX_SYSTEM_ID_MIN` and `BOX_SYSTEM_ID_MAX` constants, not on `struct space`. Please move it to `schema_def.h` and make it inline.

Reply:
Good point! Done.